src/library_noderawfs.js

```diff
@@ -82,12 +82,9 @@ mergeInto(LibraryManager.library, {
         // this stream is created by in-memory filesystem
         return VFS.read(stream, buffer, offset, length, position);
       }
-      var seeking = true;
-      if (typeof position === 'undefined') {
-        seeking = false;
-      }
       var bytesRead = fs.readSync(stream.nfd, NODEFS.bufferFrom(buffer.buffer), offset, length, position);
-      if (!seeking) stream.position += bytesRead;
+      // update position marker when non-seeking
+      if (typeof position === 'undefined') stream.position += bytesRead;
```

Comment:
this isn't right, it should be either `X === undefined` or `typeof X === 'undefined'`

Reply:
Isn't `typeof position === 'undefined'` same as `typeof X === 'undefined'`?

Reply:
Oh you're right, sorry! I must be tired...

lgtm